diesel/src/expression/expression_methods/global_expression_methods.rs

```diff
@@ -97,8 +97,8 @@ pub trait ExpressionMethods: Expression + Sized {
     /// # fn main() {
     /// #     use self::users::dsl::*;
     /// #     let connection = establish_connection();
-    /// #     connection.execute("INSERT INTO users (name) VALUES
-    /// #         ('Jim')").unwrap();
+    /// #     try!(connection.execute("INSERT INTO users (name) VALUES
```

Comment:
Slightly different problem with these, as `main` doesn't expect to return a `Result`, so the `try!` will fail. Not sure if there's any benefit to switching to `try!` here as this line is hidden from the output (It's prefixed with '#')